tests/e2e/test_e2e_examples.py

```diff
@@ -1,6 +1,6 @@
 """This module contains end-to-end tests for examples."""
 
-from yapyang import *
+from yapyang import LeafNode, ListNode, ContainerNode, ModuleNode
```

Comment:
This test module is for our `README.md` example. The import style ensures the package `__init__` module `__all__` variable contains all the nodes used in the test module. I suspect that this change was made to suppress the linter rule warnings, can we solve it?
```suggestion
from yapyang import *
```

Reply:
This indeed was added to suppress the linter rule warnings. I agree with the suggested change, will revert it.